index.html

```diff
@@ -636,6 +636,527 @@ <h3>Criteria and Criterion</h3>
         criteria and plural sets of criteria.</p>
     </section> <!-- criteria v criterion -->
   </section> <!-- introduction -->
+  <section>
+    <h2>Registration Process</h2>
+    <p>This registry provides a public vehicle for publishing updated DID Method Rubric criteria, the DID Method Rubric
+    Registry. In order to add or update a criteria, a submitter MUST submit a modification request for this registry, as a
+    pull request on the repository where this registry is hosted, where the modification request adheres to the following
+    policies:</p>
+    <section>
+      <h3>General Requirements</h3>
+      <ol>
+      <li>If there are copyright, trademark, or any intellectual property
+      rights concerns, the addition and use MUST be authorized
+      in writing by the intellectual property rights holder under a F/RAND 
+      license. For examples criteria that use trademarked brand names, use the titles or excerpts from copyrighted works, or require patented technology to perform the evaluation.
```

Comment:
Wordsmith suggestions: "trademark, or any intellectual..." --> remove "any".
"For examples criteria that uses..." --> "An example of this issue might be content that uses..."

Reply:
Agreed.